activerecord/lib/active_record/connection_adapters/postgresql/column.rb

```diff
@@ -2,18 +2,8 @@ module ActiveRecord
   module ConnectionAdapters
     # PostgreSQL-specific extensions to column definitions in a table.
     class PostgreSQLColumn < Column #:nodoc:
-      attr_accessor :array
-
-      def initialize(name, default, cast_type, sql_type = nil, null = true, default_function = nil)
-        if sql_type =~ /\[\]$/
-          @array = true
-          super(name, default, cast_type, sql_type[0..sql_type.length - 3], null)
```

Comment:
I think, now we are setting incorrect  `sql_type` value in case of array.

```
# Earlier
@sql_type  = sql_type[0..sql_type.length - 3]

# Now
@sql_type  = sql_type
```